components/Shimmer/samples/Shimmer.md

```diff
@@ -2,7 +2,7 @@
 title: Shimmer
 author: JustinXinLiu
```

Comment:
Saw this was the commit you got the --author to work, though we should try and get it on the init commit where ever the bulk of code was imported.